vault/core.go

```diff
@@ -708,6 +708,12 @@ func (c *Core) sealInitCommon(req *logical.Request) (retErr error) {
 		return retErr
 	}
 
+	// Create an identifier for the request
+	var err error
+	req.ID, err = uuid.GenerateUUID()
```

Comment:
Although it applies here too. `buildLogicalRequest` is called in both `handleSysStepDown` and `handleSysSeal`, so by moving the UUID generation code there we don't need to duplicate it anywhere else.